src/runner/PlutoRunner.jl

```diff
@@ -1096,6 +1096,10 @@ function pretty_stackcall(frame::Base.StackFrame, linfo::Core.MethodInstance)
     end
 end
 
+function pretty_stackcall(frame::Base.StackFrame, linfo::Union{Module,Method})
```

Comment:
To be consistent with the case where `linfo.def isa Method` in `pretty_stackcall(::Base.StackFrame, ::MethodInstance)` maybe?

```suggestion
function pretty_stackcall(frame::Base.StackFrame, linfo::Method)
    sprint(Base.show_tuple_as_call, linfo.name, linfo.specTypes)
end

function pretty_stackcall(frame::Base.StackFrame, linfo::Module)
```

Reply:
oops, `Method` does not have specTypes :grimacing: